Makefile

```diff
@@ -14,7 +14,7 @@ all: audit test build
 
 .PHONY: audit
 audit:
-	nancy go.sum
+	go list -json -m all | nancy sleuth
```

Comment:
This won't block approval but I noticed in a recent PR that this command was updated to use an `-- exclude-vulnerability-file` flag. I believe this command works even without an actual ignore file. That said, if we're not using an exclusion in this repo I don't see any particular need to append the flag - but figured I'd raise just in case

Reply:
Thank you, I consciously chose not include the exclusions file. Mainly due to the reason you mention regarding not having a file and it can be added if ever it were needed. But also to avoid the normalisation of exclusions. Excluding a vulnerability should be a very last resort, wherever possible we should be fixing vulnerabilities not silencing them so an exclusion file should be a very rare thing indeed. If we must include one, we can go to the extra effort of updating the command as and when necessary.

Reply:
Yeah, that makes sense to me 👍🏽